README.md

```diff
@@ -96,19 +101,19 @@ app.get('/room/:id', roomController.index)
 app.post('/room', roomController.save)
 ```
 
-Every request and response will be logged, and the most cool part: both will have the same id. This is important because you can search for this id and get all information about your request and response.
+Every request and response will be logged, and the coolest part: both will have the same id. This is important because you can search for this id and get all information about your request and response.
 
-This id is injected in req object, then if you need to log some extra information between a request and response just do something like:
+This id is injected in the `req` object, so if you need to log some extra information between a request and response just do something like this:
 
 ```js
 logger.info('some controller information', { id: req.id })
 ```
 
 ## Examples
 
-The log-generator inside examples folder will run a nodejs application that will make request for itself every milliseconds decided by the user. The application will get the milleseconds value from an environment variable called `ESCRIBA_TIMEOUT`(3000 is the default value).
+The `log-generator` inside `examples` folder will run a Node.js application that will make a request for itself every in an interval defined by the user (in milliseconds). The application will get input value from an environment variable `ESCRIBA_TIMEOUT`(3000 is the default value, this represents 3 seconds)
```

Comment:
"The application will get an input value" or "the application will get input values"